apps/cmp.c

```diff
@@ -143,6 +146,8 @@ static int opt_revreason = CRL_REASON_NONE;
 /* credentials format */
 static char *opt_certform_s = "PEM";
 static int opt_certform = FORMAT_PEM;
+static char *opt_crlform_s = "DER";
+static int opt_crlform = FORMAT_ASN1;
```

Comment:
Why is CRL form DER by default where cert form is PEM by default? This is inconsistent.

Reply:
This is to save storage space.

https://github.com/openssl/openssl/pull/23768#discussion_r1515798678

Reply:
I am not sure this is a sufficient reason for such inconsistency. Or at least it should be documented both in the code and in the manual page why the default is different for CRLs.

Reply:
@rajeev-0 this is still unresolved.

Reply:
I added the comment for using DER as preferred way for storing CRL.